### PR TITLE
Potential fix for social buttons

### PR DESCRIPTION
**Pull request by frogcrush**
This is a potential fix for #90 and also hopefully a (slight) speed improvement for users. Also stops Facebook and Twitter from being hit every time a user visits a page.

### CODE REVIEW

**Comment by frogcrush** [comment]
image
^this is what it looks like. Looking into the link for a Snapchat button as well.

File: index.html [hunk 1]
```diff
@@ -101,12 +101,12 @@ <h3 class="widget-title">Become a member!</h3>
...
         <!--End mc_embed_signup-->
 
-        <aside id="reallysimpletwitterwidget-2" class="widget widget_reallysimpletwitterwidget"><h3 class="widget-title">Stay Connected!</h3>
-            <div class="socialbuttons col-center">  
-                <!-- Facebook Button -->
-                <div class="fb-follow" data-href="https://www.facebook.com/pages/TRUSU-Computer-Science-Club/343428029131317?fref=ts" data-colorscheme="light" data-layout="button" data-show-faces="true"></div>
-                <!-- Twitter Button -->
-                <a class="twitter-follow-button" href="https://twitter.com/trucsclub" data-show-count="false" data-lang="en">Follow @trucsclub</a>  
+        <aside id="reallysimpletwitterwidget-2" class="widget widget_reallysimpletwitterwidget">
+            <h3 class="widget-title">Stay Connected!</h3>
```
Comment (mitchhentges) [inline index.html]: Let's be consistent and move the `<h3>` to the next line for each of these

File: about.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```
Comment (mitchhentges) [inline about.html]: What happened that cause each of these `<!DOCTYPE html>` tags to change?

Comment (frogcrush) [inline about.html]: Visual Studio? lol

**Comment by mitchhentges** [comment]
Finally, and more subjectively, on mobile, it seems goofy that the icons aren't centered. Perhaps something to look into later, unless you're down 2 dunk.
screenshot_20170326-225127

File: index.html (hunk 1 above)
Comment (frogcrush) [inline index.html]: Agreed. I'd love to do some more cleanup as well. Supposedly one of our asides doesn't have an ending. More cleanup could be done in a separate PR though. I'll fix these.

**Pull request merged**